src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -146,20 +146,20 @@ export const createSnapshotBuilderCountRequest = (cohort: Cohort[]): SnapshotBui
   return { cohorts: _.map(convertCohort, cohort) };
 };
 
-export const HighlightConceptName = ({ conceptName, searchFilter }): ReactElement => {
-  const startIndex = conceptName.toLowerCase().indexOf(searchFilter.toLowerCase());
+export const HighlightSearchText = ({ columnItem, searchFilter, style = {} }): ReactElement => {
```

Comment:
You actually don't need the default here! `...undefined` renders out to be undefined, so it will still do the right thing if it isn't passed in.